src/compas/geometry/transformations/translation.py

```diff
@@ -67,6 +73,14 @@ def __init__(self, matrix=None, check=True):
                     raise ValueError('This is not a proper translation matrix.')
         super(Translation, self).__init__(matrix=matrix)
 
+    @property
+    def translation_vector(self):
+        from compas.geometry import Vector
```

Comment:
why the inlined import?

Reply:
avoid circular import. the imports of transformations precede the imports of primitives etc.